docs/content/how-to/create-a-feathub-release.md

```diff
@@ -0,0 +1,248 @@
+for how to write a new note for this release. The items listed in the release
+note should meet the following criteria:
+
+- Be appropriately classified as `Bug`, `New Feature`, `Improvement`, etc.
+- Represent noteworthy user-facing changes, such as new functionality,
+  backward-incompatible API changes, or performance improvements.
+- Have occurred since the previous release; an issue that was introduced and
+  fixed between releases should not appear in the Release Notes.
+- Have an issue title that makes sense when read on its own.
+
+Adjust any of the above properties to improve the clarity and presentation of
+the Release Notes.
+
+### Stage the artifacts and Publish the Release Note
+
+Set up a few environment variables to identify the release candidate being
+built. Start with `RC_NUM` equal to `1` and increment it for each candidate.
+
+```shell
+export RC_NUM="1"
+export TAG="release-${RELEASE_VERSION}-rc${RC_NUM}"
+```
+
+Tag and push the release commit:
+
+```shell
+git tag -a ${TAG} -m "${TAG}"
+git push <remote> refs/tags/${TAG}
+```
+
+Release manager should create a Github token with permissions to create Github
+release and upload the artifacts to the Feathub repository. After you have
+created and saved the token, create a pre-release and upload the release
+artifacts to Github with the following command:
+
+```shell
+GITHUB_TOKEN=<YOUR-TOKEN> tools/releasing/create_github_release_and_upload_artifacts.sh
+```
+
+Then, head over to the release you just created on GitHub
+(https://github.com/alibaba/feathub/releases/${TAG}) and publish the release
+note to the description of the release.
+
+### Verify the artifacts
+
+Release manager should guarantee the correctness of the release candidate in the
+following aspects:
+
+- The source codes should be able to pass all the FeatHub tests. This should
+  have been verified by FeatHub's CI on Github.
+- The [FeatHub end-to-end
+  examples](https://github.com/flink-extended/feathub-examples) should be able
+  to function correctly with the Python artifacts. This could be verified by
+  manually triggering the corresponding
+  [CI](https://github.com/flink-extended/feathub-examples/actions/workflows/build-and-test.yaml)
+  with the release artifact as the Target FeatHub Python package.
+
+### Checklist to proceed to the next step
+
+- The release commit has been tagged and pushed to Github
+- The release commit has a corresponding Github Release with all the release
+  artifacts
+- Release information properly described in the release note written in the
+  Github Release.
+- The correctness of the source code and Python artifact have been verified and
+  the verification result is available to reviewers (e.g. as a Github Action
+  link).
+
+## Vote on the release candidate
+
+After the steps above, you should share the release candidiate with the Feathub
+repository admins to request their approval on this releast candidate. The
+sharing message should contain the following information:
```

Comment:
Should we also provide reviewers the link to the release note?

Reply:
The release note should have been added to the Github Release created in steps above. I'll clarify this.